src/machine/board_pca10056.go

```diff
@@ -50,3 +50,14 @@ const (
 	SPI0_MOSI_PIN Pin = 45 // P1.13
 	SPI0_MISO_PIN Pin = 46 // P1.14
 )
+
+// USB CDC identifiers
+const (
+	usb_STRING_PRODUCT      = "Nordic nRF52840DK (PCA10056)"
+	usb_STRING_MANUFACTURER = "Nordic Semiconductor"
+)
+
+var (
+	usb_VID uint16 = 0x239A
+	usb_PID uint16 = 0x8029
+)
```

Comment:
1. Is there a reason you're re-using the VID from Adafruit?
2. It looks like the nrf52*-based boards do not use these `usb_STRING_PRODUCT` etc constants yet.

Reply:
1. see https://github.com/adafruit/Adafruit_nRF52_Arduino/blob/master/boards.txt#L419

2. That is true, I will add another commit that uses the constant.

Reply:
Actually the nrf52840 boards all do use the constants. See https://github.com/tinygo-org/tinygo/blob/feature/usb-cdc-ids/src/machine/usb_nrf52840.go#L432 and https://github.com/tinygo-org/tinygo/blob/feature/usb-cdc-ids/src/machine/usb_nrf52840.go#L440

Reply:
Oh okay, then it's fine.